requirements-dev.txt

```diff
@@ -0,0 +1,8 @@
+black==22.10.0
```

Comment:
Does the requirements file not support `>=`?

Reply:
Just noticed it does.